evm/src/test/java/org/hyperledger/besu/evm/toy/ToyAccount.java

```diff
@@ -40,7 +40,7 @@ public class ToyAccount implements EvmAccount, MutableAccount {
 
   private Address address;
   private final Supplier<Hash> addressHash =
-      Suppliers.memoize(() -> address == null ? Hash.ZERO : Hash.hash(address));
+      Suppliers.memoize(() -> address == null ? Hash.ZERO : address.addressHash());
```

Comment:
is it still need to have memoize in this case ?

Reply:
I applied it to all cases where we saw `Hash.hsah(addres))` to be consistant.